src/plugins/auto_batch/src/compiled_model.hpp

```diff
@@ -5,10 +5,11 @@
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 #pragma once
 
-#include <map>
+#include <condition_variable>
+#include <thread>
 
-#include "cpp_interfaces/impl/ie_executable_network_thread_safe_default.hpp"
-#include "ie_metric_helpers.hpp"
+#include "openvino/runtime/iasync_infer_request.hpp"
+#include "openvino/runtime/icompiled_model.hpp"
 #include "plugin.hpp"
 #include "threading/ie_thread_safe_containers.hpp"
```

Comment:
should we remove old headers?

Reply:
will sync with @songbell , auto plugin also using this, maybe we can put this class in common.